mdata/cache/accnt/if.go

```diff
@@ -20,3 +20,8 @@ type EvictTarget struct {
 	Metric string
 	Ts     uint32
 }
+
+func NewAccnt(maxSize uint64) Accnt {
+	// currently we only know NewFlatAccnt
+	return NewFlatAccnt(maxSize)
+}
```

Comment:
So what do you think would be the best thing to do? Rename this `NewAccnt()` method to something like `GetAccnt()`? Or just completely drop it and directly call `NewFlatAccnt()` from the callsite of `NewAccnt()`?